src/base.jl

```diff
@@ -221,7 +236,10 @@ size_content(s::ShortString) = s.size_content
 
 @define_integers 2048 Int2048 UInt2048
 
-for T in (UInt2048, UInt1024, UInt512, UInt256, UInt128, UInt64, UInt32)
+"""These are the default types used to for selecting the size of a ShortString"""
+const def_types = [UInt32, UInt64, UInt128, UInt256, UInt512, UInt1024, UInt2048]
```

Comment:
Why not a tuple? it is immutable
```suggestion
const def_types = (UInt32, UInt64, UInt128, UInt256, UInt512, UInt1024, UInt2048)
```